src/channel.ts

```diff
@@ -198,4 +201,37 @@ export class Channel {
     this.status = 'closed';
     this.onCommandListeners = [];
   };
+
+  /**
+   * Append some extra information to an error, appropriate for being forwarded
+   * off to an error capture service.
+   *
+   * @param e - the error to wrap
+   * @param cmd  - the command that was being sent or received when the error occurred
+   * @returns the wrapped error
+   */
+  private wrapError = (e: CrosisError, cmd: api.ICommand | null = null): CrosisError => {
```

Comment:
```suggestion
  private createError = (message: string, cmd: api.ICommand | null = null): CrosisError => {
```

curious why not this?

Also I remember sentry giving me shit when i did this for OTClient, it grouped everything as if they're 1 error lol, hence [this fingerprinting logic](https://github.com/replit/repl-it-web/blob/2b17966f02dfb6b4644262f8df9ebc3d6881bb80/app/workspace/bsod/sentry.ts#L61). Maybe`this.getExtras` is more suitable.

Reply:
Suggestion LGTM.